lib/services.js

```diff
@@ -182,6 +182,9 @@ const services = {
                 'ongoing-request': false,
                 'expiry-date': archive.restoreWillExpireAt,
             });
+            md.setUserMetadata({
+                'x-amz-meta-scal-s3-restore-attempt': undefined,
+            });
```

Comment:
* we should "set" the user metadata : what we want to is actually to _remove_ this specific user metadata
* i don't understand how this solves the problem: here the `md` object is created from scratch (line 106), so it has no user metadata ; and existing metadata (if any) are set later (line 239)... what am i missing?

Reply:
Also this function has a generic name (`metadataStoreObject`) : wondering if we should fix the problem here, or if it makes sense to handle that in "higher" level function (`createAndStoreObject`) which handles the 'copy' of the metadata?